content/4_Gold/Intro_DP.mdx

```diff
@@ -183,7 +183,13 @@ The next few modules provide examples of some classical problems, or Dynamic Pro
 		url="https://atcoder.jp/contests/dp/tasks"
 		starred
 	>
-		Some tasks are beyond the scope of Gold.
+	</Resource>
+	<Resource
+		source="CF@Codeforces"
+		title="Begginer DP Contest"
+		url="https://codeforces.com/blog/entry/86532"
+	>
+		Begginer friendly classical problems. Some tasks requires input/output files. <a href="https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice">Here</a> you can check the solutions.
```

Comment:
"Begginer" should be spelt as "beginner"

Reply:
> Maybe be consistent with the file naming [here](https://github.com/Farrius/cp/tree/master/codeforces/gym_contest/dp_div3_practice).

Sure, now the name of the files are a, b, c, d...  in order to be more clear.